include/SFML/Graphics/RenderTarget.hpp

```diff
@@ -256,6 +256,23 @@ class SFML_GRAPHICS_API RenderTarget : NonCopyable
     virtual Vector2u getSize() const = 0;
 
     ////////////////////////////////////////////////////////////
+    /// \brief Activate or deactivate the render target for rendering
+    ///
+    /// This function makes the render target's context current for
+    /// future OpenGL rendering operations (so you shouldn't care
+    /// about it if you're not doing direct OpenGL stuff).
+    /// Only one context can be current in a thread, so if you
+    /// want to draw OpenGL geometry to another render target
+    /// don't forget to activate it again.
+    ///
```

Comment:
Regarding multithreading, the documentation for `RenderWindow::setActive` seems more strong than this one. Should the user be expected to obey the following rule here as well?

> if you want to make it active on another thread you have to deactivate it on the previous thread first if it was active.